examples/with-react-intl/components/PageWithIntl.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react'
+import {IntlProvider, addLocaleData, injectIntl} from 'react-intl'
+
+// Register React Intl's locale data for the user's locale in the browser. This
+// locale data was added to the page by `pages/_document.js`. This only happens
+// once, on initial page load in the browser.
+if (typeof window !== 'undefined' && window.ReactIntlLocaleData) {
```

Comment:
I just updated to use the `accept-language` module to better co-locate the code: https://github.com/relatenow/relate/commit/5c9d95aaed2f0f1fb74e2159bd67e6f72be87ee3

This means that I don't need a custom server at all to use react-intl at this point.